README.md

```diff
@@ -20,51 +20,64 @@ Useful links
 Quick Start Guide
 --------------
 
-1. Install Rust:
-   Follow [this](https://www.rust-lang.org/tools/install) guide.
-
-
-2. Install Docker:
-   Follow [this](https://docs.docker.com/engine/install/) guide.
-
-
-3. Set up your Nitro-enabled AWS EC2 instance:
+1. Set up your Ubuntu based build system:
+    - Install Rust:
+      Follow [this](https://www.rust-lang.org/tools/install) guide.
+    - Install Docker:
+      Follow [this](https://docs.docker.com/engine/install/) guide.
+    - Install tools needed to build the linux kernel:
+      Follow [this](https://kernelnewbies.org/KernelBuild) guide.
+
+2. Set up your Nitro-enabled AWS EC2 instance:
     - Install docker on your EC2:
-      Follow step #2
+      Follow [this](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-docker.html) guide.
     - Install nitro-cli on your EC2:
       Follow [this](https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave-cli-install.html) guide.
 
 
-4. Build requisite docker images needed to run container converter
+3. Build requisite docker images needed to run container converter
    ```bash
    # Run from the root of the repository
    # build enclave-base image
    cd salmiac/docker/enclave-base
    docker build -t enclave-base .
          
    # build parent-base image
-   cd ..//parent-base
+   cd ../parent-base
    docker build -t parent-base .
     ```
 
-5. Compile container converter:
+4. Build the enclave kernel. This step takes a long time and needs to be done only once. The artifacts produced by this step need not be cleaned up unless the kernel config is updated.
+   ```bash
+   cd amzn-linux-nbd
+   ./build-enclave-kernel.sh build
+    ```
+
+5. Build the converter image. To produce a debug build of the converter, ensure the release flag is removed from the step below.
     ```bash
       # Run from the root of the repository
       cd salmiac
-      ./build-converter.sh   
+      # To produce a debug build of the converter, ensure the release flag is removed from the step below.
+      ./build-converter.sh --release
```

Comment:
Building this needs following packages:
```
sudo apt install \
    pkg-config \
    libclang-dev \
    cmake \
    libpcap-dev
```

Reply:
Are you sure we need those tools to build the converter? Rust should be enough to build the converter.

Reply:
Code in build scripts of some dependencies needed these.